lessons/2-methods.md

```diff
@@ -128,6 +129,143 @@ puts add_two(3)
 
 </details>
 
-## Homework
+### Homework
+
+[CodeWars - Can we divide it?](https://www.codewars.com/kata/5a2b703dc5e2845c0900005a/train/ruby)
+
+# German
+
+## Lesson 2 - Methods
+
+  - [DataTypen Fortsetzung](#datatypen-fortsetzung)
+    - [Boolean](#boolean)
+  - [Methoden](#methoden)
+    - [Allgemein](#allgemein)
+    - [Rückgabewerte](#rückgabewertreturn-value)
```

Comment:
The secition is actually called `Rückgabewert` (singular). I'd suggest to adjust the name here:

```suggestion
    - [Rückgabewert](#rückgabewertreturn-value)
```